Gemfile

```diff
@@ -44,7 +44,7 @@ end
 
 group :development do
   gem 'byebug', platforms: :ruby
-  gem 'rubocop', '1.28.0'
+  gem 'rubocop', '1.51.0'
```

Comment:
https://docs.rubocop.org/rubocop/compatibility.html#support-matrix

This can be any version currently. You can make this open ended, and we can add a constraint later.